src/components/organisms/BottomNav.js

```diff
@@ -25,6 +25,13 @@ const StyledBottomNavigation = styled(BottomNavigation)`
   .MuiBottomNavigationAction-label.Mui-selected {
     font-size: 12px;
   }
+  @media (max-width: 380px) {
```

Comment:
any reason not using the breakpoints from `import { bps } from "@/ui/theme" ?

Reply:
No idea about the `bps` thing and it's settings. changes welcome :)